indigo_app/views/tasks.py

```diff
@@ -64,7 +65,8 @@ def get(self, request, *args, **kwargs):
 
         # initial state
         if not params.get('state'):
-            params.setlist('state', ['open', 'assigned', 'pending_review', 'blocked'])
+            pass
```

Comment:
The idea behind this is we don't want to show completed tasks by default, otherwise it would spam the listing.